rfqmath/convert.go

```diff
@@ -62,6 +62,10 @@ func MilliSatoshiToUnits[N Int[N]](milliSat lnwire.MilliSatoshi,
 // compute the total amount of mSAT (X) as follows:
 //   - X = (U / Y) * M
 //   - where M is the number of mSAT in a BTC (100,000,000,000).
+//
+// TODO(ffranr): This function only works with BigInt as the underlying
```

Comment:
We only want to use the big int version. The other was added just to have another implementation, and also demonstrate where the unit64 will lose precision.

Reply:
With this TODO, in a later PR, I hope to make `UnitsToMilliSatoshi` specific to `BigInt` fixed-points by removing the generic type parameter.